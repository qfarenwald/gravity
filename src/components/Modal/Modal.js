import React from 'react';
import { Link } from 'react-router-dom';
import './Modal.css';

export const Modal = () => {
  return (
    <section className="Modal">
      <h3>YOUR SHUTTLE IS BOOKED</h3>
      <h3>A CONFIRMATION EMAIL HAS BEEN SENT</h3>
      <h3>GET READY TO DROP IN!</h3>
      <Link to='/trails' className="link"><h5>BACK TO TRAILS</h5></Link>
    </section>
  )
}
