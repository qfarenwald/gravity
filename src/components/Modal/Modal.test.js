import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { Modal } from './Modal';

describe('Modal', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Modal />);

    expect(wrapper).toMatchSnapshot();
  });
});
