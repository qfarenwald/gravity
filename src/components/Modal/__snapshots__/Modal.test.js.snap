// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should match snapshot 1`] = `
<section
  className="Modal"
>
  <h3>
    YOUR SHUTTLE IS BOOKED
  </h3>
  <h3>
    A CONFIRMATION EMAIL HAS BEEN SENT
  </h3>
  <h3>
    GET READY TO DROP IN!
  </h3>
  <Link
    className="link"
    to="/"
  >
    <h5>
      BACK TO LOCATIONS
    </h5>
  </Link>
</section>
`;
